include/server.h

```diff
@@ -56,7 +56,9 @@ namespace kiwix
        void setNbThreads(int threads) { m_nbThreads = threads; }
        void setVerbose(bool verbose) { m_verbose = verbose; }
        void setTaskbar(bool withTaskbar, bool withLibraryButton)
-         { m_withTaskbar = withTaskbar; m_withLibraryButton = withLibraryButton; }
+        { setTaskbar(withTaskbar, withLibraryButton, m_blockExternalLinks); }
+       void setTaskbar(bool withTaskbar, bool withLibraryButton, bool blockExternalLinks)
```

Comment:
I dislike the idea of changing the signature of `setTaskbar`, especially to add an argument not related to taskbar.

It would be simpler to add a new method `setBlockExternalLink`.

Reply:
done